rethinkdb/metadata.csv

```diff
@@ -0,0 +1,44 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+rethinkdb.server.total,gauge,,node,,Number of connected servers in the cluster.,0,rethinkdb,Server total
+rethinkdb.database.total,gauge,,,,Number of databases in the cluster.,0,rethinkdb,Database total
+rethinkdb.database.table.total,gauge,,table,,Number of tables in a given database.,0,rethinkdb,Database table total
+rethinkdb.table.secondary_index.total,gauge,,index,,Number of secondary indexes in a given table.,0,rethinkdb,Table secondary index total
+rethinkdb.stats.cluster.queries_per_sec,gauge,,query,,Number of queries executed in a cluster per second.,0,rethinkdb,Stats cluster queries per sec

```

Comment:
```suggestion
rethinkdb.stats.cluster.queries_per_sec,gauge,,query,second,Number of queries executed in a cluster per second.,0,rethinkdb,Stats cluster queries per sec
```

Same for all other `*_per_sec` metrics.

Reply:
Added to the RFC tables, will update here when regenerating metadata :+1:

Reply:
Done in 1f2896a0